application/templates/cronjob.yaml

```
@@ -31,10 +31,19 @@ spec:
 {{ end }}
   jobTemplate:
     spec:
+{{- with $job.activeDeadlineSeconds }}
+      activeDeadlineSeconds: {{ . }}
+{{- end }}
       template:
         metadata:
           labels:
           {{- include "application.labels" $ | nindent 12 }}
+          {{- with $job.additionalPodLabels }}
+          {{ toYaml . | nindent 12 }}
```

```suggestion
          {{- toYaml . | indent 12 }}
```
I think `{{- include "application.labels" $ | nindent 12 }}` also has an unnecessary N in Nindent.